docs/installation.md

```diff
@@ -0,0 +1,118 @@
+  `kubectl -n arlon create secret generic argocd-creds --from-file /tmp/config`
+- Delete the temporary config file
+- Clone the arlon git repo and cd to its top directory
+- Create the CRDs: `kubectl apply -f config/crd/bases/`
+- Deploy the controller: `kubectl apply -f deploy/manifests/`
+- Ensure the controller eventually enters the Running state: `watch kubectl -n arlon get pod`
+
+## Arlon CLI
+
+Download the CLI for the [latest release](https://github.com/arlonproj/arlon/releases/latest) from GitHub.
+Currently, Linux and MacOS operating systems are supported.
+Uncompress the tarball, rename it as `arlon` and add to your PATH
+
+Run `arlon verify` to check for prerequisites.
+Run `arlon install` to install any missing prerequisites.
+
+The following instructions are to manually build CLI from this code repository.
+
+### Building the CLI
+
+- Clone this repository and pull the latest version of a branch (main by default)
+- From the top directory, run `make build`
+- Optionally create a symlink from a directory
+  (e.g. `/usr/local/bin`) included in your ${PATH} to the `bin/arlon` binary
+  to make it easy to invoke the command.
+
+## Cluster orchestration API providers
+
+Arlon currently supports Cluster API on AWS cloud. It also has experimental
+support for Crossplane on AWS.
+
+### Cluster API
+
+Using the [Cluster API Quickstart Guide](https://cluster-api.sigs.k8s.io/user/quick-start.html)
+as reference, complete these steps:
+
+- Install `clusterctl`
+- Initialize the management cluster.
+  In particular, follow instructions for your specific cloud provider (AWS in this example)
+  Ensure `clusterctl init` completes successfully and produces the expected output.
+
+### Crossplane (experimental)
+
+Using the [Upbound AWS Reference Platform Quickstart Guide](https://github.com/upbound/platform-ref-aws#quick-start)
+as reference, complete these steps:
+
+- [Install UXP on your management cluster](https://github.com/upbound/platform-ref-aws#installing-uxp-on-a-kubernetes-cluster)
+- [Install Crossplane kubectl extension](https://github.com/upbound/platform-ref-aws#install-the-crossplane-kubectl-extension-for-convenience)
+- [Install the platform configuration](https://github.com/upbound/platform-ref-aws#install-the-platform-configuration)
+- [Configure the cloud provider credentials](https://github.com/upbound/platform-ref-aws#configure-providers-in-your-platform)
+
+You do not need to go any further, but you're welcome to try the Network Fabric example.
+
+FYI: *we noticed the dozens/hundreds of CRDs that Crossplane installs in the management
+cluster can noticeably slow down kubectl, and you may see a warning that looks like*:
+
+```shell
+I0222 17:31:14.112689   27922 request.go:668] Waited for 1.046146023s due to client-side throttling, not priority and fairness, request: GET:https://AA61XXXXXXXXXXX.gr7.us-west-2.eks.amazonaws.com/apis/servicediscovery.aws.crossplane.io/v1alpha1?timeout=32s
+```
+### Automatic setup
```

Comment:
automatic setup is a v0.10+ feature

Reply:
Added a note saying so. If someone is still on v0.3 out there, it would be good motivation to upgrade. :)